provider/cmd/pulumi-resource-aws-native/schema.json

```diff
@@ -57538,30 +57543,55 @@
                     "value": "ap-southeast-2"
                 },
                 {
-                    "name": "CaCentral",
```

Comment:
Meh, seems like we accidentally forgot the `1` at the end when we created that enum.
I'm gonna manually add this one and deprecate it